src/exports.js

```diff
@@ -47,7 +47,6 @@ export { default as LiveHash } from "./account/LiveHash.js";
 export { default as LiveHashAddTransaction } from "./account/LiveHashAddTransaction.js";
 export { default as LiveHashDeleteTransaction } from "./account/LiveHashDeleteTransaction.js";
 export { default as LiveHashQuery } from "./account/LiveHashQuery.js";
-export { default as NetworkName } from "./NetworkName.js";
```

Comment:
Can't remove this unfortunately, it'd be a breaking change. If you'd like you can just export whatever this was exporting here directly without re-creating the file.